Neos.ContentRepository.TestSuite/Classes/Behavior/Features/Bootstrap/Features/NodeCreation.php

```diff
@@ -94,6 +94,9 @@ public function theCommandCreateRootNodeAggregateWithNodeIsExecutedWithPayloadAn
     public function theEventRootNodeAggregateWithNodeWasCreatedWasPublishedToStreamWithPayload(TableNode $payloadTable)
     {
         $eventPayload = $this->readPayloadTable($payloadTable);
+        if (!isset($eventPayload['workspaceName'])) {
+            $eventPayload['workspaceName'] = 'some-workspace';
+        }
```

Comment:
I used this in all the "The event [...] was published with payload" steps to fill the `workspaceName` if it wasn't specified (we already do that for dimension space points etc).

An alternative would be to adjust the behat tests themselves.. not sure about it

Reply:
auto fill seems fine. dont we do this with the csid as well?

Reply:
We used to fall back to currentContentStreamId and could do the same with currentWorkspaceName. I'm Not too fond of autofill Here and would prefer an Exception If neither ist given

Reply:
oh lol sorry i missed that we literally hardcode `'some-workspace'` here. I was under impression we would fallback to `currentWorkspaceName` which i also prefer ;)

Reply:
it's because it does not matter in the cases. But I agree that we should be specific here, I'll adjust the tests and require the workspace name to be set

Reply:
phew, that was tedious but I think I adjusted all affected tests (let's see what the CI says)
I love our tests, but IMO we test too much from the _event_s perspective (there are 83 hits for `/the event .* was published with payload/`). IMO we should mostly execute commands and expect events and/or state as a result (except for the rare low-level cases that are not feasible to recreate with commands).. But that's for another time :)

Reply:
Yes i have an issue as reminder for that: https://github.com/neos/neos-development-collection/issues/4336

but i was under the impression that most things now use commands already.